examples/angular-cli/src/stories/core/parameters/bootstrap-options.stories.ts

```diff
@@ -20,5 +20,3 @@ export default {
 } as Meta;
 
 export const WithPreserveWhitespaces: Story = (_args) => ({});
-
-WithPreserveWhitespaces.storyName = 'With Preserve Whitespaces';
```

Comment:
Oh, nice! You even made it work to catch the formatted case, not just the literal. 💯  👏

Reply:
Thank you!